src/SFML.Graphics/VertexArray.cs

```diff
@@ -111,13 +112,14 @@ public void Clear()
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Resize the vertex array
-        /// 
-        /// If \a vertexCount is greater than the current size, the previous
+        /// </summary>
+        /// <remarks>
+        /// If vertexCount is greater than the current size, the previous
```

Comment:
```suggestion
        /// If <see cref="VertexCount"/> is greater than the current size, the previous
```

Reply:
Great catch

Reply:
I don't know that this works/has a purpose.  The <param> tag provides information about the vertexCount argument.

Reply:
Upon further investigation, it appears that <paramref name="vertexCount" /> would be the correct method of doing this.  This tag could be used in a LOT of places.